src/bun.js/api/bun/subprocess.zig

```diff
@@ -1642,7 +1642,7 @@ pub const Subprocess = struct {
 
                     if (argv0 == null) {
                         var path_buf: [bun.MAX_PATH_BYTES]u8 = undefined;
-                        const resolved = Which.which(&path_buf, PATH, arg0.slice()) orelse {
+                        const resolved = Which.which(&path_buf, PATH, cwd, arg0.slice()) orelse {
```

Comment:
should this `cwd` be the one on line 1709?

Reply:
this was from reverting https://github.com/oven-sh/bun/pull/9691. i'll look if this is correct code.